Eshop.WebApi/Controllers/ProductController.cs

```diff
@@ -16,41 +24,46 @@ public class ProductController : Controller
         [HttpGet]
         public List<Product> GetProducts()
         {
-            return Products;
+            return dbContext.Products.ToList();
         }
 
         [HttpGet("{id}")]
         public Product GetProduct(int id)
         {
-            return Products.First(x => x.Id == id);
+            return dbContext.Products.First(x => x.Id == id);
         }
 
         [HttpPost]
         public Product CreateProduct(string title, string description, decimal price, int categoryId)
         {
             var category = CategoryController.Categories.First(x => x.Id == categoryId);
-            var newId = Products.Max(x => x.Id) + 1;
+            var newId = dbContext.Products.Max(x => x.Id) + 1;
```

Comment:
check category for Post request it should be exactly same, just line 39 to fetch category will stay

Reply:
not sure if its right, but fixed ig